Assets/js/project-leads.js

```diff
@@ -0,0 +1,28 @@
+const projectLeads = ()=>{
+    document.addEventListener('DOMContentLoaded', function() {
+        const showMoreButton = document.querySelector('.show-more-button');
+        const experts = document.querySelectorAll('.expert');
+        let visibleExpertsCount = 5;
```

Comment:
@dkdeepesh  Store this value in data-attribute in markup and get the value from there
Cc:- @jvaishnavrp @mathrik